qiskit/transpiler/target.py

```diff
@@ -525,7 +509,7 @@ def update_from_instruction_schedule_map(self, inst_map, inst_name_map=None, err
                         continue
                 try:
                     # Update gate error if provided.
-                    props.error = error_dict[inst_name][qargs]
+                    setattr(props, "error", error_dict[inst_name][qargs])
```

Comment:
Yes, there should be no difference, this was old code that behaved based on having `InstructionProperties` be wrapped in a `Py` reference. This will be promptly moved back to its original syntax.